lib/screenplay/alerts/cache/store.ex

```diff
@@ -0,0 +1,64 @@
+defmodule Screenplay.Alerts.Cache.Store do
```

Comment:
Do we need a second module/process for this? It's not storing any state or providing significant parallelism, so it seems like we could inline this code into the other module to simplify things.

Reply:
That's a fair point. Managed to consolidate while keeping test coverage at 100%.